portfolio/src/main/java/com/google/sps/comment/CommentStorage.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.comment;
+
+import com.google.sps.config.Constants;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.PreparedQuery;
+import java.util.ArrayList;
+import java.text.SimpleDateFormat;  
+import java.util.Date;
+import java.lang.NumberFormatException;
+
+/** Wrapper over Datastore. Allows for safe adding and retrieval of entities, requiring
```

Comment:
[nit: start the text of a Javadoc comment on the next line after the opening `/**`](https://google.github.io/styleguide/javaguide.html#s7-javadoc), here and below